ember-shortcuts.js

```diff
@@ -60,56 +60,77 @@
     }
   }
 
-  function makeDispatch(router, filters) {
-    function triggerShortcut(def, event) {
-      var i, action, handler, infos;
+  function filter(filters, event) {
+    for (var i = 0; i < filters.length; i++) {
+      if (!filters[i](event)) return false;
+    }
+    return true;
+  }
+
+  function triggerEvent(filters, event, kc, callback) {
```

Comment:
While I agree, I originally didn't rename it because the existing code used `kc` so often, and I didn't want to bloat the PR. I've done a full rename here: cac6d65